Makefile

```diff
@@ -1,11 +1,32 @@
 REPO_ROOT = $(dir $(realpath $(firstword $(MAKEFILE_LIST))))
 
+.PHONY: help
+help:
+	@echo "Usage:"
+	@echo "  make lint-all                  Lint all charts in the 'charts' directory."
+	@echo "  make validate                  Validate the PR."
+	@echo "  make docs-all                  Generate helm docs for all charts in the 'charts' directory."
+	@echo "  make docs-<chart_name>         Generate helm docs for a specific chart, e.g. 'make docs-castai-agent'."
```

Comment:
[suggestion]: no need to maintain manual help target. You can use something below(old example from one of my repos -- [src](https://github.com/aweris/github-actions-runner/blob/master/Makefile))


```make
.PHONY: install-tools
install-tools: ## Install tools
install-tools: vendor ; $(info $(M) installing tools)
	$(Q) $(GOINSTALL) $(shell cat tools.go | grep _ | awk -F'"' '{print $$2}')

.PHONY: help
help: ## Shows this help message
	$(Q) echo 'usage: make [target] ...'
	$(Q) echo
	$(Q) echo 'targets : '
	$(Q) echo
	$(Q) fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'| column -s: -t
	$(Q) echo
```

Reply:
Added dynamic help target